packages/fleet/fleet-agent/generated-changes/overlay/Chart.yaml.orig

```diff
@@ -0,0 +1,13 @@
+annotations:
```

Comment:
I want to highlight that in the `dev-v2.7` branch this is a patch, not an overlay. But I *think* this is fine.